thoth/python/source.py

```diff
@@ -160,22 +165,6 @@ def _gather_hashes(self, path: str) -> list:
                         })
         return digests
 
-    def _get_digests(self, artifact_url: str, artifact_name: str) -> list:
```

Comment:
@KPostOffice This method is also used in **_download_artifacts_sha** just in case. I liked how you combined them into one, nice.